external/include/opus/opus_types.h

```diff
@@ -0,0 +1,177 @@
+/******************************************************************
```

Comment:
same question as I mentioned above,  opus_xxx.h
If we modified something then we can add copyright, but this is external folder, we don't have to put our copyright in c or cpp or h file.